lib/phoenix_live_view/controller.ex

```diff
@@ -36,6 +36,8 @@ defmodule Phoenix.LiveView.Controller do
 
   """
   def live_render(%Plug.Conn{} = conn, view, opts) do
+    ensure_layout_set(conn)
```

Comment:
I don't think we need this code now. :D Since now it is guaranteed we set the layout from router, if there iss no layout here, it is likely on purpose.